cmd/connectivity_pod_to_url.go

```diff
@@ -32,7 +31,7 @@ func newConnectivityPodToURLCmd() *cobra.Command {
 				return errors.Errorf("provide both SOURCE_POD and DESTINATION_URL")
 			}
 
-			fromPod, err := kuberneteshelper.PodFromString(args[0])
+			fromPod, err := pod.FromString(args[0])
```

Comment:
fromPod -> srcPod

Reply:
Also, for consistency, in L47 also changing `fromPod` to `srcPod` 
https://github.com/openservicemesh/osm-health/blob/main/cmd/connectivity_pod_to_url.go#L47